src/test/regress/expected/rpt.out

```diff
@@ -1241,6 +1241,40 @@ select c from rep_tab where c in (select distinct d from rand_tab);
  2
 (2 rows)
 
+set optimizer= off;
+drop table if exists t;
+NOTICE:  table "t" does not exist, skipping
+create table t (i int) distributed replicated;
+insert into t select 1;
+create or replace function f(i int) returns int language sql security definer as $$ select i; $$;
+explain SELECT (select f(i) from t);
```

Comment:
You may use `costs off` to decrease patch footprint in this case. More, I think that explain analyze is enough (valid plan + SIGSEGV absence).

Reply:
done